spinner.go

```diff
@@ -137,34 +148,14 @@ func (s *Spinner) Reverse() {
 
 // Color will set the struct field for the given color to be used
 func (s *Spinner) Color(c string) error {
+	var colorError error
```

Comment:
ah you're right it wouldn't be, I just like to have 1 return statement per method/function if I can help it, I can update it if you feel that way is better, both make sense to me